hicommonwealth/commonwealth
# Add migration to create groups from token thresholds

## Link to Issue
Closes: #5396

## Description of Changes
- Adds migration that generates groups from token thresholds
- Group names are formatted as `[token] Holders`

## "How We Fixed It"


## Test Plan
- Run migration via `yarn migrate-db`
- Open stargate members page: http://localhost:8080/stargatetoken/members?tab=groups
  - Confirm that a group called "STG Holders" is created
- To perform global refresh, make request to `POST http://localhost:8080/api/refresh-all` as super admin
  - with body: { "jwt": ... }
  - Confirm that logs show community refresh status and "done" message when finished


## Deployment Plan
Run migration

## Other Considerations
N/A
